ios/RNGoogleCast/RNGoogleCast.m

```diff
@@ -265,6 +271,72 @@ - (void)stopObserving {
   }
 }
 
+
+RCT_REMAP_METHOD(getCurrentMedia,
+                 getCurrentMediaResolver: (RCTPromiseResolveBlock) resolve
+                                rejecter: (RCTPromiseRejectBlock) reject) {
+
+  dispatch_async(dispatch_get_main_queue(), ^{
+    if (self->castSession == nil) {
```

Comment:
Should this code be wrapped in a try{} that `rejects` the promise (or squashes the exception and calls `resolve` if that's what we'd want) if there is a sync error thrown by this code, or will RCT_REMAP_METHOD wrapper handle converting sync errors into Promise rejections for us?